<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dial_control_par</class>
 <widget class="QDialog" name="dial_control_par">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>192</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Характеристики объекта и контроля</string>
  </property>
  <widget class="QLabel" name="label_thick_units">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>40</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>мм</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_diam">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>14</y>
     <width>145</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Диаметр трубы</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_diam_units">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>16</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>мм</string>
   </property>
  </widget>
  <widget class="ParamLineEdit" name="ed_thick">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>37</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_thick">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>37</y>
     <width>145</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Толщина стенки трубы</string>
   </property>
  </widget>
  <widget class="ParamLineEdit" name="ed_diam">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>14</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_thick_lim_neg">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>82</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>допуск толщины &quot;–&quot;</string>
   </property>
  </widget>
  <widget class="ParamLineEdit" name="ed_thick_lim_neg">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>81</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_thick_lim_neg_units">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>84</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>мм</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_thick_lim_pos">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>105</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>допуск толщины &quot;+&quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_thick_lim_pos_units">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>107</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>мм</string>
   </property>
  </widget>
  <widget class="ParamLineEdit" name="ed_thick_lim_pos">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>104</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>201</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Применить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox_relative_thick">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Относительный допуск</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ParamLineEdit</class>
   <extends>QLineEdit</extends>
   <header>paramlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ed_diam</tabstop>
  <tabstop>ed_thick</tabstop>
  <tabstop>ed_thick_lim_neg</tabstop>
  <tabstop>ed_thick_lim_pos</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>dial_control_par</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>dial_control_par</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
